contributors/devel/conformance-tests.md

```diff
@@ -1,110 +1,150 @@
+# Option A: run all conformance tests serially
+kubetest --provider=skeleton --test --test_args="--ginkgo.focus=\[Conformance\]"
 
-# ... and finish up with remaining tests in serial
-go run hack/e2e.go -- --provider=skeleton --test --test_args="--ginkgo.focus=\[Serial\].*\[Conformance\]"
+# Option B: run parallel conformance tests first, then serial conformance tests serially
+GINKGO_PARALLEL=y kubetest --provider=skeleton --test --test_args="--ginkgo.focus=\[Conformance\] --ginkgo.skip=\[Serial\]"
+kubetest --provider=skeleton --test --test_args="--ginkgo.focus=\[Serial\].*\[Conformance\]"
 ```
 
-### Kubernetes Conformance Document
-For each Kubernetes release, a Conformance Document will be generated
-that lists all of the tests that comprise the conformance test suite, along
-with the formal specification of each test. For example conformance document for 
-1.9 can be found [here](https://github.com/cncf/k8s-conformance/blob/master/docs/KubeConformance-1.9.md).
-This document will help people understand what features are being tested without having to look through
-the testcase's code directly.
+## Kubernetes Conformance Document
 
+For each Kubernetes release, a Conformance Document will be generated that lists
+all of the tests that comprise the conformance test suite, along with the formal
+specification of each test. For an example, see the [v1.9 conformance doc].
+This document will help people understand what features are being tested without
+having to look through the testcase's code directly.
 
-## Adding New Tests
 
-To promote a testcase to the conformance test suite, the following
-steps must be taken:
-- as a prerequisite, the test case is already part of e2e and is not flaky 
-- the testcase must use the `framework.ConformanceIt()` function rather
-  than the `framework.It()` function
-- the testcase must include a comment immediately before the
-  `framework.ConformanceIt()` call that includes all of the required
-  metadata about the test (see the [Test Metadata](#test-metadata) section)
-- use "Promote xxx e2e test to Conformance" as template of your PR title
-- tag your PR with "/area conformance" label
-- send your PR to Sig-Architecture for review by adding "@kubernetes/sig-architecture-pr-reviews" 
-also CC the relevant Sig and Sig-Architecture
+## Promoting Tests to Conformance
 
+To promote a test to the conformance test suite, open a PR as follows:
+- is titled "Promte xxx e2e test to Conformance"
+- includes information and metadata in the description as follows:
+  - "/area conformance" on a newline
+  - "@kubernetes/sig-architecture-pr-reviews @kubernetes/sig-foo-pr-reviews
+    @kubernetes/cncf-conformance-wg" on a new line, where sig-foo is whichever
+    sig owns this test
+  - any necessary information in the description to verify that the test meets
+    [conformance test requirements], such as links to reports or dashboards that
+    prove lack of flakiness
+- modifies code as follows:
```

Comment:
Let’s add a bullet “- contains no other modifications to test source code” - that’s been very useful.